tfx/components/example_gen/base_example_gen_executor_test.py

```diff
@@ -279,10 +279,10 @@ def testInvalidFeatureBasedPartitionWithProtos(self):
     # Run executor.
     example_gen = TestExampleGenExecutor()
     with self.assertRaisesRegexp(
-        RuntimeError, 'Split by `partition_feature_name` is only supported for '
-        'FORMAT_TF_EXAMPLE payload format.'):
+        RuntimeError, 'Split by `partition_feature_name` is only supported '
+        'for FORMAT_TF_EXAMPLE or FORMAT_TF_SEQUENCE_EXAMPLE payload format.'):
       example_gen.Do({}, self._output_dict, self._exec_properties)
 
 
 if __name__ == '__main__':
-  tf.test.main()
+  tf.test.main()
```

Comment:
don't remove the last line (lint/format error), same apply to other files

Reply:
I see, I will keep this in mind